migreringer/src/main/resources/db/migration/defaultDS/3.1/V3.1_01__TFP-5571-lokalt_oppgave_tabelle.sql

```diff
@@ -21,7 +23,9 @@ create table OPPGAVE
 
 comment on table OPPGAVE is 'Inneholder oppgaver som skal løses av saksbehandlere i journalføring.'
 /
-comment on column OPPGAVE.OPPGAVE_TYPE is 'Type av oppgaven f.eks JFR for journalføring. Mappes til Oppgavetype kodeverk.'
+comment on column OPPGAVE.TYPE is 'Type av oppgaven f.eks JFR for journalføring. Mappes til Oppgavetype kodeverk.'
+/
+comment on column OPPGAVE.STATUS is 'Status av oppgaven. Foreløpig AAPENT, FERDIGSTILT.'
```

Comment:
Opprettet og ferdigstilt?

Reply:
Like greit å bruke samme koder som Gosys .... enklere mapping

Reply:
Problemet er at jeg ikke helt vett når de endrer status - men det ser ut at etter man oppretter så blir det AAPENT (det bruker vi i søket). Og så blir det FERDIGSTILLT etter saken er fikset. Har ikke tenkt å sette flere status hos oss.

Btw. kan det være at vi ikke viser alle oppgaver i løsningen? Om status endres til UNDER_BEHANDLING så vil vi ikke få treff i ny løsning. Hva tenker du @AnjaAalerud ?

@jolarsen ja jeg har tenkt å bruke samme dto som mot gosys.

Reply:
har jeg misforstått kanskje? Jeg  trodde vi skulle "fri" oss fra Gosys for manuell journalføring? At vi ikke oppretter oppgavene i Gosys men hos oss

Når det gjelder under_behandling - er det når vi henter oppgaver i gosys? Jeg er ikke sikker på tilstand til en slik oppgave som er under_behanlding i gosys..

Reply:
@mrsladek tenkte at det er greit å legge seg nær OppgaveDto som går til/fra frontend - derfor bruke ytelsetype framfor behandling_tema som gosys har. 
Det er 2 alternativ - ligge nær Dto til/fra frontend med egen mapping, eller nær GosysDto og bruke lik  mapping.
Vi kan nok forenkle mest dersom bruker alternativ 1

Reply:
@AnjaAalerud ja dette ville være absolut det beste men da mister vi de oppgaver som ble "flyttet" over til tema FOR av andre team (pga de ble feil opprettet på et annet tema). Så vi må nok lese inn ting fra gosys. Men som du sier det blir veldig få til slutt.